contrib/cirrus/pr-should-link-jira

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Intended for use in CI, for -rhel branches: check git PR, barf if no jira links
+#
+
+ME=$(basename $0)
+
+set -e
+
+# Github label which allows overriding this check
+OVERRIDE_LABEL="No Jira Link"
+
+# Only -rhel branches need jira links
```

Comment:
Don't we need main also?  That eventually turns into a Jira branch.

Reply:
Not sure what you mean by that. There are tons of changes in main that don't have a redhat jira issue to link to?